### DISCUSSION TITLE
Svelte support for eslint-plugin-testing-library?

### DISCUSSION THREAD

**Discussion by hslee2008** [testing-library/eslint-plugin-testing-library#751] [Q&A] [answered]
I am not sure if I am missing something here, but is Svelte not supported?
There seems to be no documentation nor any indication at all in general.
Currently, I had no choice but to turn of linting in my test files for Svelte project.

Please help!

**Comment by Belco90** [comment, marked as answer]
Hey @hslee2008! This ESLint plugin hasn't been tested for Svelte Testing Library. It might happen everything works fine, but each Testing Library has some particular and unique behaviors we need to address. That's the reason we don't provide official support or a preset config. If you test it and see it's compatible, please let us know or even create a PR to include it as a supported Testing Library framework!

**Reply by Belco90** [reply]
For example, I see [`fireEvent` is async in Svelte Testing Library](https://testing-library.com/docs/svelte-testing-library/api#fireevent-async). This is similar to Vue Testing Library, where using the `no-await-sync-events` doesn't make sense, and we recommend enabling `await-fire-event` instead.
